pages/linux/esearch.md

```diff
@@ -0,0 +1,13 @@
+# esearch
+
+> Esearch performs a new Entrez search using terms in indexed fields.
```

Comment:
```suggestion
> Esearch performs a new Entrez search using terms in indexed fields (It is part of the edirect package).
```
Is this fine?

Reply:
I separated it out and so technically there are three sentences in the description.  One of the sentences says it is part of the edirect package.